test/parallel/test-os.js

```diff
@@ -101,7 +101,7 @@ console.error(interfaces);
 switch (platform) {
   case 'linux':
     {
-      const filter = function(e) { return e.address == '127.0.0.1'; };
+      const filter = function(e) { return e.address === '127.0.0.1'; };
       const actual = interfaces.lo.filter(filter);
```

Comment:
Nit: Can the `filter` be replaced with an arrow function, still within 80 chars limits?